### PR TITLE
SIG Autoscaling Annual Report 2021

### LINKED ISSUE #5498: 2021 Annual Report: SIG Autoscaling
Chairs: @gjtempleton @mwielgus 
Liasion: @derekwaynecarr

- [ ] Consult SIG to complete draft of report
- [ ] If needed, consult with Steering Committee liasion on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your SC liasion for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, SC will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by March 1st, 2021
- PR should be reviewed and merged by April 1st, 2021

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liasion directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig autoscaling
/committee steering

### PR DESCRIPTION
**Pull request by gjtempleton**
**Which issue(s) this PR fixes**:

Fixes #5498 

cc: @mwielgus

### CODE REVIEW

**Review by mwielgus — APPROVED** [review]
/lgtm

**Comment by gjtempleton** [comment]
Pending review by SC liason
/hold

**Comment by dims** [comment]
@derekwaynecarr this is on hold for you to review, PTAL.

File: sig-autoscaling/annual-report-2021.md
Comment (cblecker) [inline sig-autoscaling/annual-report-2021.md:16]: Why aren't recordings being made?

File: sig-autoscaling/annual-report-2021.md
Comment (mrbobbytables) [inline sig-autoscaling/annual-report-2021.md:42]: Not a blocker by any means, but this is something we can potentially help you in ContribEx. We've had success with various mentoring cohorts and other means to get people onboarded into areas that need help 👍

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Few small things, thanks for putting this together!

File: sig-autoscaling/annual-report-2021.md
Comment (gjtempleton) [reply sig-autoscaling/annual-report-2021.md:42]: Thanks, since this PR was put together and off the back of discussions around the annual report process in the SIG Chairs meeting I've been having some discussions with @LappleApple around the enhancements liaison process she's been starting.

There's definitely room for expanding this with the help of ContribEx, particularly in seeking to grow the pool of code reviewers and approvers, but focused on making this first step successful for now.

File: sig-autoscaling/annual-report-2021.md
Comment (derekwaynecarr) [inline sig-autoscaling/annual-report-2021.md:50]: is there a pipeline of folks that have been identified or expressed an interest to help in this area?

File: sig-autoscaling/annual-report-2021.md
Comment (gjtempleton) [reply sig-autoscaling/annual-report-2021.md:16]: A good question. 

I don't think we've made recordings of the meetings for as long as I've been involved in the SIG. I've added discussing beginning to make recordings and make them available to the agenda for the next SIG office hours. I'm assuming the [guidelines here](https://github.com/kubernetes/community/blob/master/communication/youtube/youtube-guidelines.md) are the go-to reference for setting this up?

File: sig-autoscaling/annual-report-2021.md
Comment (gjtempleton) [reply sig-autoscaling/annual-report-2021.md:50]: It varies depending on the area. 

There's a number of engaged contributors in the VPA space helping to drive this forward.

The cluster autoscaler generally has engaged cloud provider owners, but the core code is generally contributed to by fewer people with fewer current obvious candidates to progress through the contributor ladder.

The HPA has a number of developers who have previously contributed large areas of new functionality, or who are currently expressing interest in helping to graduate features currently in alpha or beta. As we're currently putting together the plan for finally graduating the v2 APIs from beta, this presents a natural opportunity to try to mentor a number of contributors who have expressed interest in this work.

File: sig-autoscaling/annual-report-2021.md
Comment (cblecker) [reply sig-autoscaling/annual-report-2021.md:16]: Yup, that's the right one!

Also for reference, providing publicly recorded meetings is a [requirement](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#sig-roles-and-organizational-governance) of all sigs. If you have questions or concerns about doing this in principle, please come talk to Steering!

If it's just a matter of getting things set up from a technical standpoint to enable this, come talk to contribex :)

**Comment by gjtempleton** [comment]
@derekwaynecarr Is there any more feedback on this/things we need to change/update here?

**Comment by mrbobbytables** [comment]
/lgtm
with steering hat on 👍

**Pull request merged**
